airgun/widgets.py

```diff
@@ -1441,3 +1443,23 @@ def read(self):
         if self.is_completed:
             return self.MESSAGE.read()
         return self.TASK.read()
+
+
+class PieChart(GenericLocatorWidget):
+    """Default Pie Chart that can be found across application. At that moment
+    only return values that displayed inside of the chart
+    """
+    chart_title_text = Text(
+        ".//*[name()='svg']//*[name()='tspan']"
+        "[contains(@class,'donut-title-small-pf')]"
+    )
+    chart_title_value = Text(
+        ".//*[name()='svg']//*[name()='tspan']"
+        "[contains(@class,'donut-title-big-pf')]"
+    )
+
+    def read(self):
+        """Return dictionary that contains chart title name as key and chart
+        value as its value
+        """
+        return {self.chart_title_text.read(): self.chart_title_value.read()}
```

Comment:
Think chart_title_text and chart_title_value can target different meanings in the chart, and they should be returned separately,
```python
return dict(title=self.chart_title_text.read(), title_value=self.chart_title_value.read())
```  
or let it without this read function and we should have chart_title_text and chart_title_value returned by the default read.

Reply:
actually on a chart it looks like 'Success: 100%' in the center of a chart, so previous statement target that idea, but not a problem, I can do it with `dict()` idea. Default read is not good in that case

Reply:
@oshtaier the 100% can have nothing to do with Success
for example it can be Failed : 100%   the 100% is related action applied to how many hosts.
the action was 100% so action was applied to all hosts
overall action status is Failed but for some hosts it can be Success

Reply:
okay, it doesn't make sense to change that method at that moment, as it will just add necessity to add more assert statement